client/app/bundles/HelloWorld/constants/helloWorldConstants.jsx

```diff
@@ -0,0 +1,13 @@
+// See https://www.npmjs.com/package/mirror-creator
+// Allows us to set up constants in a slightly more concise syntax. See:
+// client/app/bundles/HelloWorld/actions/helloWorldActionCreators.jsx
+import mirrorCreator from 'mirror-creator';
```

Comment:
remove `mirrorCreator`

Instead, use the simple form: 

```
export default {
  HELLO_WORLD_NAME_UPDATE: 'HELLO_WORLD_NAME_UPDATE'
};
```

So this file just has above lines..

@alexfedoseev Please confirm